apps/studio/components/interfaces/Database/Replication/DestinationPanel.tsx

```diff
@@ -404,7 +405,7 @@ const DestinationPanel = ({
                                     <Input_Shadcn_
                                       {...field}
                                       type="number"
-                                      {...form.register('maxFillSecs', {
+                                      {...form.register('maxFillMs', {
```

Comment:
small nit: `form.register('maxFillMs')` is the same as `field` in these props so it's not needed.